src/components/search-list/search-list.html

```diff
@@ -1,47 +1,58 @@
 <!-- Generated template for the SearchListComponent component -->
 <div class="container-width">
+  
+  <!---- Search Bar ---->
   <ion-searchbar
     [(ngModel)]="searchInput"
     [showCancelButton]="shouldShowCancel"
     (ionInput)="doSearch($event)"
     (ionCancel)="onCancel($event)">
   </ion-searchbar>
   
-  <div *ngIf="displayDropdown">
-    <ion-card>
-      <ion-label>Search results</ion-label>
+  <!---- Search Results (exists only when search bar is clicked)  ---->
+  <div *ngIf="displayDropdown" style="position:fixed; z-index: 105;" class="container-width">
+    <ion-card style="margin-top: 0px;">
+      <ion-label padding-left="16px">Search results</ion-label>
       <ion-list>
       <ion-item class="item" *ngFor="let obj of filteredList" (click)="selectedItem($event, obj)">
         <h2> {{ obj.name }} </h2>
-        <h3> Taken on: <strong>{{ obj.date }}</strong> </h3>
+        <h3> Taken on: <strong>{{ obj.date | date:fullDate }}</strong> </h3>
       </ion-item>
       </ion-list>
     </ion-card>
   </div>
+  
+  <ion-label text-center> <strong>Tests I've taken</strong></ion-label>
 
-  <div>
-    <ion-label text-center> <strong>Tests I've taken</strong></ion-label>
-    <ion-label *ngIf="emptyHistory" text-center> 
-      <p> No tests taken yet </p>
-      <p> Click <u class="light-text" (click)="goToTestsPage()">here</u> to take a test </p> 
-      <p> <br></p>
-      <p (click)="emptyHistory=!emptyHistory"><u><i>View Mock Data</i></u></p> 
-    </ion-label>
-    <ion-list *ngIf="!emptyHistory">
-      <ion-item class="item display-block" *ngFor="let obj of ourList" (click)="selectedItem($event, obj)">
-        <div id="text-block" class="half-container">
-          <h2> <strong> {{ obj.name }} </strong> </h2>
-          <h3> Taken on: <strong>{{ obj.date }}</strong> </h3>
-        </div>
-        
-        <div class="small-chart half-container">
-          <mini-graph class="chart-align"></mini-graph>
-        </div>
+  <!---- Empty history View ---->
+  <ion-label *ngIf="emptyHistory" text-center> 
+    <p> No tests taken yet </p>
+    <p> Click <u class="light-text" (click)="goToTestsPage()">here</u> to take a test </p> 
+    <p> <br></p>
+    <p (click)="emptyHistory=!emptyHistory"><u><i>View Mock Data</i></u></p> 
+  </ion-label>
   
-      </ion-item>
-    </ion-list>
-  </div>
+ <!---- Display our test history if it is not empty ---->
+  <ion-list *ngIf="!emptyHistory">
+    <!---- Each Test Card (test info and mini chart) ---->
+    <ion-item class="item display-block" *ngFor="let obj of ourList | testsByTime : searchTime " (click)="selectedItem($event, obj)">
```

Comment:
We can chain pipes here so ourList can be filtered by the search value from the search bar and the date as well.
for example
 *ngFor="let obj of ourList | searchTermPipe: searchTerm | testsByTime : searchTime